Extensions/RosettaConsole/Console.cpp

```diff
@@ -286,7 +287,28 @@ void Console::ModifyDeck()
 
     OperateDeck(selectedDeck);
 }
+void Console::AddDeckCode()
+{
+    std::cout << "========================================\n";
+    std::cout << "          Add Deck With Code!           \n";
+    std::cout << "========================================\n";
 
+    std::cout << "Input the deck code to add :";
+
+    std::string DeckCode;
+    std::cin >> DeckCode;
+    DeckInfo NewDeck;
+    try
+    {
+        NewDeck = ::RosettaStone::PlayMode::DeckCode::Decode(DeckCode);
+    }
+    catch (const std::runtime_error& error)
+    {
+        std::cerr << error.what() << '\n';
+        return;
+    }
+    OperateDeck(m_account->GetNumOfDeck());
+}
```

Comment:
```suggestion
    std::string deckCode;
    std::cin >> deckCode;
    
    DeckInfo newDeck;
    try
    {
        newDeck = ::RosettaStone::PlayMode::DeckCode::Decode(deckCode);
    }
    catch (const std::runtime_error& error)
    {
        std::cerr << error.what() << '\n';
        return;
    }
    
    OperateDeck(m_account->GetNumOfDeck());
}
```

Reply:
Thank you for pointing out. I expected that `clang-format` would have pointed out if I would have mistaken these kinds of indentation and newline. Would that be the case? - If so, I would need to check my `git-hooks` again, in case I did something wrong there. Anyway, thank you for your time for the review.

Reply:
You need to correct the first letter of the local variable to be lowercase such as `deckCode` and `newDeck`.